src/actions/rootRequestedThunk.tsx

```diff
@@ -23,15 +23,12 @@ export const rootRequestedThunk = () => {
         const handleTheData = (dbrecords: any) => {
             const convert = (dbrecord: IDatabaseRecord) => {
                 return {
-                    parent:         -1,
+                    parent:         dbrecord.childof,
```

Comment:
ok, so for the record, the ``dbrecord.childof`` for the top node will be ``-1`` in the database by changes in the preprocessing repository [here](https://github.com/NLeSC-Storyteller/query-builder-preprocessing/tree/b41f5067e9daea143d8c7a58bc946704dd890993).